docs/reST/ref/math.rst

```diff
@@ -195,6 +195,36 @@ Multiple coordinates can be set using slices or swizzling
 
       .. ## Vector2.distance_squared_to ##
 
+   .. method:: move_towards
+
+      | :sl:`returns a vector that moves toward to the given vector at the given distance.`
+      | :sg:`move_towards(Vector2, float) -> Vector2`
```

Comment:
Rather than have having args as `Vector2` and `float` which are actually typenames, we need to specify the parameter names here. I'm thinking something like "target" and "distance" will do

Reply:
Oh wait NVM, we need to indicate types here because everywhere else in the math module it's done this way